guides/source/active_record_migrations.md

```diff
@@ -220,88 +243,89 @@ class RemovePartNumberFromProducts < ActiveRecord::Migration[8.0]
-  end
-end
+```bash
+$ bin/rails generate migration AddUserRefToProducts user:belongs_to
 ```
 
-This migration will create a `user_id` column. [References](#references) are a
-shorthand for creating columns, indexes, foreign keys, or even polymorphic
-association columns.
+generating a migration that looks like the same as above.
 
-There is also a generator which will produce join tables if `JoinTable` is part of the name:
+There is also a generator which will produce join tables if `JoinTable` is part
+of the name:
 
 ```bash
-$ bin/rails generate migration CreateJoinTableCustomerProduct customer product
+$ bin/rails generate migration CreateJoinTableUserProduct user product
 ```
 
 will produce the following migration:
 
 ```ruby
-class CreateJoinTableCustomerProduct < ActiveRecord::Migration[8.0]
+class CreateJoinTableUserProduct < ActiveRecord::Migration[8.0]
   def change
-    create_join_table :customers, :products do |t|
-      # t.index [:customer_id, :product_id]
-      # t.index [:product_id, :customer_id]
+    create_join_table :users, :products do |t|
+      # t.index [:user_id, :product_id]
+      # t.index [:product_id, :user_id]
     end
   end
 end
 ```
 
-[`add_column`]: https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_column
-[`add_index`]: https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_index
-[`add_reference`]: https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference
-[`remove_column`]: https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-remove_column
+[`add_column`]:
+    https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_column
+[`add_index`]:
+    https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_index
+[`add_reference`]:
+    https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference
+[`remove_column`]:
+    https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-remove_column
 
-### Model Generators
+### Other Generators that Create Migrations
 
-The model, resource, and scaffold generators will create migrations appropriate for adding
-a new model. This migration will already contain instructions for creating the
-relevant table. If you tell Rails what columns you want, then statements for
-adding these columns will also be created. For example, running:
+In addition to the `migration` generator, the `model`, `resource`, and
+`scaffold` generators will create migrations appropriate for adding a new model.
+This migration will already contain instructions for creating the relevant
+table. If you tell Rails what columns you want, then statements for adding these
```

Comment:
I'd recommend avoiding phrases like "tell Rails" and instead use phrasing like "supply parameters to the generator".

Reply:
Disagree here.  I agree that "tell Rails" might lead to the question of "how?", but we want the Guides to be reader-friendly, and certainly new-developer friendly. We don't need to strip the it of all colloquialism.

Reply:
Or may be something like this:

```suggestion
table. If you tell Rails (pass additional details) what columns you want, then statements for adding these
```

Reply:
I can understand the desire to use colloquial verbs like "tell" to create a more gentle style, especially for less experienced folks. 

I guess it depends on the audience, as mentioned.

Phrasing like "supply" or "provide" parameters might be more technical and less friendly for newcomers, but more conventional for experienced programmers.

Sounds like the colloquial style is preferred here.